clang/lib/DPCT/APINames.inc

```diff
@@ -358,7 +358,7 @@ ENTRY(cudaGraphInstantiateWithParams, cudaGraphInstantiateWithParams, false, NO_
 
 
 // C++ high level API functions of runtime API
-ENTRY(cudaOccupancyMaxPotentialBlockSize, cudaOccupancyMaxPotentialBlockSize, false, NO_FLAG, P0, "no direct mapping yet", "NA")
+ENTRY(cudaOccupancyMaxPotentialBlockSize, cudaOccupancyMaxPotentialBlockSize, true, NO_FLAG, P0, "no direct mapping yet", "NA")
```

Comment:
need file the warnning message. DPCT1111

Reply:
done